doc/api/cli.md

```diff
@@ -3070,6 +3070,9 @@ one is included in the list below.
 * `--experimental-shadow-realm`
 * `--experimental-specifier-resolution`
 * `--experimental-strip-types`
+* `--experimental-test-coverage`
```

Comment:
I haven't tested, but I would expect passing `--experimental-test-coverage` in `NODE_OPTIONS` to cause problems.

Reply:
Why?

Reply:
The test runner explicitly tries to [filter](https://github.com/nodejs/node/blob/94c327c753b386f8aef9754b18b6714396d1d718/lib/internal/test_runner/runner.js#L99) that argument out when launching worker processes, but `NODE_OPTIONS` sidesteps that. It may work, but I wouldn't count on it.

Reply:
Thanks! I'll test that locally against https://github.com/nodejs/userland-migrations/pull/7, which should spawn workers via test runner.